app/views/layouts/_cookie_consent.html.haml

```diff
@@ -12,10 +12,7 @@
       },
       cookie: {
         name: "_swapmyvote_cookie_consent",
-        domain: ".swapmyvote.uk",
-        domain: ".swapmyvote.herokuapp.com",
-        domain: ".swapmyvotedev.herokuapp.com",
-        domain: "localhost"
+        domain: "#{ENV['SERVER_HOST']}"
```

Comment:
This might cause a regression without the `.` prefix - @egglestn what do you think?

Reply:
The . prefix is recommended for subdomains, and I think also helps with older browser support. If we don't have any subdomains, it probably works, I don't have a strong opinion on whether we keep it or not - would have to test to be sure.

Reply:
We only use `www.swapmyvote.uk` and `swapmyvotedev.herokuapp.com` - no sub-domains, so I guess we're good here.